srcpkgs/kget/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'kget'
+pkgname=kget
+version=18.04.3
+revision=1
+build_style=cmake
+configure_args="-DDESKTOPTOJSON_EXECUTABLE=/usr/bin/desktoptojson"
+hostmakedepends="extra-cmake-modules python qt5-qmake qt5-host-tools
+ pkg-config kdoctools kconfig kcoreaddons"
+makedepends="qt5-devel qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql
```

Comment:
Please shorten with xmindep.

Reply:
How so? xmindep queries the remote repository. Since the remote repository doesn't contain the package, the result is empty.

Reply:
xmindep qt5-devel … libktorrent-devel